Core/Workflow/Step.py

```diff
@@ -44,7 +43,7 @@ def __init__( self, step_type = None, obj = None, parent = None ):
       self.parameters = ParameterCollection( self, obj.parameters )
       self.module_instances = InstancesPool( self, obj.module_instances )
       if obj.module_definitions != None:
```

Comment:
if not obj.module_definitions ?
Or if it really needs to be compared to None
if obj.module_definitions is not None ?